packages/react-native-editor/__device-tests__/gutenberg-editor-lists.test.js

```diff
@@ -2,28 +2,38 @@
  * Internal dependencies
  */
 import { blockNames } from './pages/editor-page';
-import { backspace, isAndroid } from './helpers/utils';
+import { backspace, isAndroid, isLocalEnvironment } from './helpers/utils';
 
 describe( 'Gutenberg Editor tests for List block', () => {
 	// Prevent regression of https://github.com/wordpress-mobile/gutenberg-mobile/issues/871
 	it( 'should handle spaces in a list', async () => {
 		await editorPage.addNewBlock( blockNames.list );
-		let listBlockElement = await editorPage.getBlockAtPosition(
-			blockNames.list
-		);
-		// Click List block on Android to force EditText focus
-		if ( isAndroid() ) {
-			await listBlockElement.click();
-		}
+		let listBlockElement = await editorPage.getListBlock();
 
 		// Send the list item text.
-		await editorPage.sendTextToListBlock( listBlockElement, '  a' );
+		await editorPage.typeTextToTextBlock( listBlockElement, '  a', false );
 
 		// Send an Enter.
-		await editorPage.sendTextToListBlock( listBlockElement, '\n' );
+		await editorPage.typeTextToTextBlock( listBlockElement, '\n', false );
+
+		// Click List block on Android.
+		// Only needed when testing in local environment
+		if ( isAndroid() && isLocalEnvironment() ) {
+			await listBlockElement.click();
+		}
```

Comment:
I'm curious about why this is only needed in the local environment 🤔 , shouldn't we expect the same behavior in both? It might actually be an issue diverging the testing logic between both platforms as it would make it hard for debugging errors.

Reply:
Yeah this is a weird one (considering I'm using the exact same emulators on both envs 🙃  ), and that's a valid concern, we might have to think about more platforms if this condition is added (iOS vs Android vs Local vs CI). 

Let me try again if this can be removed so we don't complicate the condition.

Reply:
Great, thanks for checking it 🙇. Ideally, we should avoid having different logic between local and remote environments as both should behave equally. The reality is that we've experienced that each has its own nuances, so it might happen that there's no other way than to apply these types of tweaks.

Reply:
Resolving this conversion as I see that this comment was addressed in https://github.com/WordPress/gutenberg/pull/40670/commits/4f6917ea29be33315f89b96ea811278408390207.